app/controllers/items_controller.rb

```diff
@@ -43,6 +43,18 @@ def create
     redirect_to trip_path(@trip)
   end
 
+  def update
```

Comment:
should we use another name for the method as 'update' is part of Rails? @antondgx